interactive/__init__.py

```diff
@@ -34,3 +36,5 @@ class Analysis:
     time_value: str
     title: str
     id: str | None = None  # noqa: A003
+    start_date: str = dates.START_DATE
```

Comment:
I think this is fine for the moment, but might need to be passed in from the form once that's updated rather than be in two different places.